lib/util/files/index.js

```diff
@@ -30,3 +30,13 @@ module.exports = function createFile(filepath, contents) {
 		delete LOCKS[filepath];
 	});
 };
+
+function mkdirpSync(directory) {
```

Comment:
There's no need to expose this for others to use, is there?

Reply:
No.